opm/simulators/utils/PartiallySupportedFlowKeywords.cpp

```diff
@@ -229,6 +229,13 @@ partiallySupported()
                {5,{true, allow_values<std::string> {"F"}, "SPECGRID(TYPE): only option F (Cartesian grids supported) supported"}}, // COORD_TYPE
             },
          },
+         {
+            "SLAVES",
+            {
+                {1,{true, [](const std::string& val){ return val.size()<=8;}, "SLAVES(SLAVE_RESERVOIR): Only names of slave reservoirs of up to 8 characters are supported."}},
```

Comment:
> We should check with the engineers whether they really only use 8 or reky on truncation and use the rest for further documentation. I think the commercial simulator just truncates.

@blattms Referring to: https://github.com/OPM/opm-simulators/pull/5453#discussion_r1746986481 : then we should use "false" instead of "true" here and in the keyword handlers in opm-common, we should truncate the length if the slave reservoir if it is more than 8 characters long?